dist/flatpak/com.github.input-leap.input-leaps.yml

```diff
@@ -10,34 +10,59 @@
 # https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1013
 
 app-id: com.github.inputleap.InputLeap
-runtime: org.gnome.Platform
-runtime-version: '42'
-sdk: org.gnome.Sdk
-command: input-leap-flatpak
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
```

Comment:
why are you using such an old version ? We want to build with Qt6 This is why its default.

Reply:
the input-leap GUI is qt5 as far as I can tell

Reply:
I stand corrected, there is a qt6 flag. However, when I compiled the qt6 version, the GUI executable just returns and the GUI doesn't appear. If you can compile it against qt6 and it works, let me know and we can figure out what is missing.

Reply:
I stand corrected. The bug was my own machine not allowing background flatpaks. I've updated it to qt6.